src/Ide/Plugin/Ormolu.hs

```diff
@@ -36,24 +43,24 @@ descriptor plId = (defaultPluginDescriptor plId)
 -- ---------------------------------------------------------------------
 
 provider :: FormattingProvider IO
-provider _lf ideState typ contents fp _ = do
+provider lf ideState typ contents fp _ = withIndefiniteProgress lf title Cancellable $ do
```

Comment:
I guess we should have these for all formatting providers?

Reply:
In many other plugins too, specially in the Eval plugin